### PR TITLE
Finally, a variable for the text from dry firing! (blowpipe text and sound tweak too)

### PR DESCRIPTION
**Pull request by brugged**
## About The Pull Request
makes it so you have a cool custom variable for the text displayed for dry firing, and makes the blowgun not make a gun-clicking noise and go *click*. Instead its silent and goes *pshoo*

## Why It's Good For The Game
now the blowgun is less stupid. also now anything you want can have any custom dry firing text. good for adding non-trigger projectile weapons.

## Changelog
:cl:
tweak: blowpipe no longer makes a phantom trigger click sound when empty
refactor: custom dry fire text given to the masses
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 1]
```diff
@@ -116,6 +116,8 @@
 	icon_state = "blowgun"
 	item_state = "blowgun"
 	fire_sound = 'sound/items/syringeproj.ogg'
+	dry_fire_sound = null //dont got a trigger, no dry fire sound
+	dry_fire_text = "pshoo" //heehee pshoo
```
Comment (Superyodeler) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:120]:
```suggestion
	dry_fire_sound = null //dont got a trigger, no dry fire sound	Wasp Edit - Dry firing
	dry_fire_text = "pshoo" //heehee pshoo		Wasp Edit - Dry firing
```

File: code/modules/projectiles/gun.dm [hunk 2]
```diff
@@ -146,7 +147,7 @@
 	return TRUE
 
 /obj/item/gun/proc/shoot_with_empty_chamber(mob/living/user as mob|obj)
-	to_chat(user, "<span class='danger'>*click*</span>")
+	to_chat(user, "<span class='danger'>*[dry_fire_text]*</span>")
```
Comment (Superyodeler) [inline code/modules/projectiles/gun.dm:150]:
```suggestion
	to_chat(user, "<span class='danger'>*[dry_fire_text]*</span>")		// Wasp Edit - Dry firing
```

File: code/modules/projectiles/gun.dm [hunk 3]
```diff
@@ -23,6 +23,7 @@
 	var/vary_fire_sound = TRUE
 	var/fire_sound_volume = 50
 	var/dry_fire_sound = 'sound/weapons/gun/general/dry_fire.ogg'
+	var/dry_fire_text = "click"				//change this on non-gun things
```
Comment (Superyodeler) [inline code/modules/projectiles/gun.dm:26]:
```suggestion
	var/dry_fire_text = "click"				//change this on non-gun things		Wasp Edit - Dry firing
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Works fine, just needs margular comments

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 1 above)
Comment (brugged) [reply code/modules/projectiles/guns/misc/syringe_gun.dm:120]: oh yeah probably a good idea to comment things that aren't just //heehee pshoo

**Review by Superyodeler — APPROVED** [review]

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 1 above)
Comment (Superyodeler) [reply code/modules/projectiles/guns/misc/syringe_gun.dm]: heehee pshoo is a very important comment

**Pull request merged**